unitrade-backend/src/main/java/ca/mcgill/ecse428/unitrade/unitradebackend/controller/PersonRestController.java

```diff
@@ -53,4 +58,78 @@ public ResponseEntity<PersonResponseDto> getPerson(@PathVariable("id") Long id)
         return new ResponseEntity<PersonResponseDto>(
                 PersonResponseDto.createDto(personService.getPerson(id)), HttpStatus.OK);
     }
+
+    @PreAuthorize("hasRole('USER')")
```

Comment:
The `@PreAuthorize("hasRole('USER')")` annotation can be applied at the class level as a global permission, the same way as the `@ApiResponses` is applied at the class level. For `createPerson(...)`, you can annotate it with `@PreAuthorize("permitAll()")` to override the permission of the class and set it to permit any access.